algorithms/indexing/symmetry.py

```diff
@@ -1,321 +1,22 @@
 from __future__ import absolute_import, division, print_function
 
-import copy
 import logging
-import math
 
-import dials.util
+from six.moves import StringIO
```

Comment:
patch incoming

Reply:
_should_ work. Haven't run it through with an appropriate example triggering the code path though.